plugins/BEdita/Core/src/Model/Action/SignupUserAction.php

```diff
@@ -356,14 +361,13 @@ protected function checkExternalAuth(array $data)
      *
      * @param string $url OAuth2 provider URL
      * @param string $accessToken Access token to use in request
+     * @param array $options OAuth2 request options
      * @return array Response from an OAuth2 provider
      * @codeCoverageIgnore
      */
-    protected function getOAuth2Response(string $url, string $accessToken): array
+    protected function getOAuth2Response(string $url, string $accessToken, array $options = []): array
     {
-        $response = (new Client())->get($url, [], ['headers' => ['Authorization' => 'Bearer ' . $accessToken]]);
-
-        return (array)$response->getJson();
+        return (new OAuth2())->response($url, $accessToken, $options);
```

Comment:
Myabe I would pass the configuration in the class constructor, but anyway it's ok for me